### ISSUE TITLE
Ignored RAML issue: Invalid datetime format value

### ISSUE THREAD

**Issue by postatum** [aml-org/amf#470]
### Issue
Running code below with each of the RAML inputs provided should fail or report validation error because according to RAML 1.0 spec provided RAML inputs are invalid.

The spec says:
> A timestamp in one of the following formats: if the format is omitted or set to rfc3339, uses the "date-time" notation of RFC3339; if format is set to rfc2616, uses the format defined in RFC2616.

Spec link:
https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#date

### amf-client-js version
3.1.9

### RAML Inputs
```raml
#%RAML 1.0
title: Some API

types:
  SomeType:
    type: datetime
    format: asdasdasd
```

### JavaScript Code
```js
const amf = require('amf-client-js')

const ramlStr = ''  // Pick one of the above RAML inputs

async function main () {
  await amf.AMF.init()
  const model = await amf.AMF.raml10Parser().parseStringAsync(ramlStr)
  let report
  try {
    report = await amf.AMF.validate(
      model, amf.ProfileNames.RAML10, amf.MessageStyles.RAML)
  } catch (e) {
    console.log(e.toString())
  }
  if (!report.conforms) {
    report.results.map(res => {
      console.log(`${res.level} - ${res.message}`)
    })
  }
}

main()
```

**Comment by BlockLucas** [comment]
https://www.mulesoft.org/jira/browse/APIMF-1518

**Comment by gutee** [comment]
This is fixed and will be released in amf 3.5.0
